src/main/java/com/user/management/data/user/dao/impl/UserDaoImpl.java

```diff
@@ -215,11 +204,11 @@ public User updatePassword(User user) {
         LOGGER.debug("Updating password failed.");
         return user;
     }
+
     @Override
     public String getPasswordByUsername(String username) {
-        String selectPasswordQuery = "SELECT password FROM login WHERE username=?";
         try (Connection connection = ConnectionHelper.getConnection();
-             PreparedStatement preparedStatement = connection.prepareStatement(selectPasswordQuery)) {
+             PreparedStatement preparedStatement = connection.prepareStatement(GET_PASSWORD_BY_USERNAME_STATEMENT)) {
```

Comment:
fix the try-with-resource. remove preparedstatement from try()

Reply:
okay napo sir